### ISSUE TITLE
Accessibility violation: The ARIA attributes “aria-hidden” are not valid for the element <style> with ARIA role “none”

### ISSUE THREAD

**Issue by joyful-these** [ReactTooltip/react-tooltip#823]
Reopening issue #728.

Accessibility violation is detected on the <style> element generated by `react-tooltip`, described as: `The ARIA attributes “aria-hidden” are not valid for the element <style> with ARIA role “none”. ARIA attributes must be valid for the element and ARIA role to which they are assigned.`

Using the latest `react-tooltip` version, `4.5.0`.

<img width="687" alt="Screen Shot 2022-11-18 at 3 07 43 PM" src="https://user-images.githubusercontent.com/108295261/202794775-a19567bb-8eb6-4279-a3d3-bcb99364d1ae.png">
<img width="570" alt="Screen Shot 2022-11-18 at 3 08 03 PM" src="https://user-images.githubusercontent.com/108295261/202794887-097ca10a-e66d-4406-8711-497c83080739.png">
<img width="570" alt="Screen Shot 2022-11-18 at 3 08 50 PM" src="https://user-images.githubusercontent.com/108295261/202794893-4dbe4ea5-7f9b-4242-81ff-7988d2e7afc6.png">

**Comment by danielbarion** [comment]
Hi @joyful-these, thanks for reporting the problem!

It's an already know problem, this one will not exist on V5, but V5 will probably take some time to be released, you can follow the progress here: https://github.com/ReactTooltip/react-tooltip/pull/820

Also, if you want and can, please do a PR to fix this problem in V4 and I'll review it as soon as possible.

Thanks!

**Comment by danielbarion** [comment]
Closing all v4 issues due to V5 released.

If necessary, please open a new issue using V5, thanks!
